starter-code/iteration-2/main.ts

```diff
@@ -1,15 +1,12 @@
-// Add the reference to the "TodoInterface"
-import * as todoInterface from './todoInterface.js';
+import {TodoInterface}  from './todoInterface';
 
-class Todo implements todoInterface{
-  arr:Array<string> = [];
-  addTask(e:string):number;
-  listAllTasks();
-  deleteTask(e:string):number;
+class Todo implements TodoInterface{
+constructor(  arr:Array<string>,
+  addTask(e:string):number,
+  listAllTasks(),
```

Comment:
Se haría: 
class Todo implements TodoInterface {
  todoList: Array<string> = [];

  addTask(task: string): number {
    this.todoList.push(task);

    console.log("=========== NEW INSERTION ===========");
    console.log(`Task "${task}" inserted in the list`);

    return this.todoList.length;
  }